ext/frc42_dispatch/Cargo.toml

```diff
@@ -1,19 +1,18 @@
 [package]
-name = "frc42_dispatch"
+name = "recall_frc42_dispatch"
```

Comment:
I had to publish the packages in `/ext` since the versions on crates do not match what we have here.